src/components/call-to-action-overlay/call-to-action-overlay.tsx

```diff
@@ -57,7 +57,7 @@ const CallToActionOverlay = withText({closeLabel: 'overlay.close'})(
   connect(mapStateToProps)(
     ({title, description, buttons, onClose, onClick, closeLabel, descriptionLines, sizeClass}: CallToActionOverlayProps): any => {
       return (
-        <div data-testid="call-to-action-overlay" className={`${styles.callToActionOverlay} ${sizeClass}`}>
+        <div role="alert" data-testid="call-to-action-overlay" className={`${styles.callToActionOverlay} ${sizeClass}`}>
```

Comment:
As I understand `<CallToActionOverlay />` renders dynamically, does screen-readers handles changes for such nodes properly?

Reply:
the screen reader notifies when the overlay appears and reads out the title, and when the overlay shows again with a different title it reads the new title. this covers the issue, as i understandv it